app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -591,13 +603,38 @@ export default class Heatmap {
 
   sortColumns(direction) {
     this.columnClustering = null;
+    this.gColumnDendogram.classed(cs.shouldSortColumns, true);
     orderBy(this.columnLabels, label => label.label, direction).forEach(
       (label, idx) => {
         label.pos = idx;
       }
     );
   }
 
+  // Re-sorts the rows. The rendered order of rows is determined solely by
+  // the `pos` property of each rowLabel and filteredRowLabel.
+  sortRows(direction) {
+    this.rowClustering = null;
+    this.gRowDendogram.classed(cs.shouldSortRows, true);
+    orderBy(
```

Comment:
Why do you need to sort this? Wouldnt it suffice to sort the filtered?

Reply:
I'll remove that sort.

Reply:
Will remove .